src/main/java/core/db/RowMapper.java

```diff
@@ -0,0 +1,8 @@
+package core.db;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public interface RowMapper {
```

Comment:
이 인터페이스는 제네릭 자료형으로 만들어야하지 않을까요?

```suggestion
public interface RowMapper<T> {
```